### PR TITLE
x11-themes/kde-plasma-chili: new package

### PR DESCRIPTION
**Pull request by beatussum**
The hottest login theme around for **KDE Plasma 5** (**sddm**).
Chili is hot, just like a real chili! Spice up the login experience for your users, your family and yourself. Chili reduces all the clutter and leaves you with a clean, easy to use, login interface with a modern yet classy touch.

### CODE REVIEW

File: x11-themes/kde-plasma-chili/kde-plasma-chili-0.5.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+DESCRIPTION="The hottest login theme around for KDE Plasma 5"
+HOMEPAGE="https://github.com/MarianArlt/kde-plasma-chili"
+SRC_URI="https://github.com/MarianArlt/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+RDEPEND="
+	kde-plasma/plasma-meta
```
Comment (juippis) [inline x11-themes/kde-plasma-chili/kde-plasma-chili-0.5.0.ebuild]: What a weird requirement.

File: x11-themes/kde-plasma-chili/kde-plasma-chili-0.5.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+	x11-misc/sddm
+"
+
+DOCS=(
```
Comment (juippis) [inline x11-themes/kde-plasma-chili/kde-plasma-chili-0.5.0.ebuild]: These should all be part of default `einstalldocs` array, so no need to specify them here (please do test it though).

File: x11-themes/kde-plasma-chili/kde-plasma-chili-0.5.0.ebuild [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+	AUTHORS
+	CHANGELOG.md
+	CREDITS
+	README.md
+)
+
+src_install() {
+	einstalldocs
+
+	rm "${S}/${DOCS[@]}" "${S}/LICENSE.md"
```
Comment (juippis) [inline x11-themes/kde-plasma-chili/kde-plasma-chili-0.5.0.ebuild]: I don't think this is necessary, if you don't use globbing to install the theme below. If you can't work around it, append with `|| die` and preferably do this operation in `src_prepare` phase.

File: x11-themes/kde-plasma-chili/kde-plasma-chili-0.5.0.ebuild [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+	dodir "/usr/share/sddm/themes/${PN}"
+	cp -pPR "${S}"/* "${D}/usr/share/sddm/themes/${PN}" || die
```
Comment (juippis) [inline x11-themes/kde-plasma-chili/kde-plasma-chili-0.5.0.ebuild]: You can use `insinto` and `doins` instead, no need to `die` with those. Also, `S` should be the CWD already.
(@gyakovlev)

File: x11-themes/kde-plasma-chili/kde-plasma-chili-0.5.0.ebuild (hunk 2 above)
Comment (beatussum) [reply x11-themes/kde-plasma-chili/kde-plasma-chili-0.5.0.ebuild]: Yes, except **CHANGELOG.md**.

File: x11-themes/kde-plasma-chili/kde-plasma-chili-0.5.0.ebuild (hunk 3 above)
Comment (beatussum) [reply x11-themes/kde-plasma-chili/kde-plasma-chili-0.5.0.ebuild]: It has to be run after _einstalldocs_: and therefore in _src_install_.

**Pull request closed without merging**